### PR TITLE
dark mode

### PR DESCRIPTION
**Pull request by adityakush24**
# Description
- [x] Source branch in your fork has meaningful name (not `main`)

Fixes Issue: 
https://github.com/beautify-web/js-beautify/issues/2057

# Before Merge Checklist 
These items can be completed after PR is created.

(Check any items that are not applicable (NA) for this PR)

- [NA] JavaScript implementation
- [NA] Python implementation (NA if HTML beautifier)
- [NA] Added Tests to data file(s)
- [NA] Added command-line option(s) (NA if
- [NA] README.md documents new feature/option(s)

# Screenshot
image

### CODE REVIEW

**Comment by okyanusoz** [comment]
We could also add a Switch to Dark/White mode option.

**Comment by vrindavan** [comment]
> We could also add a Switch to Dark/White mode option.

So is that set to the system theme at the moment?

**Comment by okyanusoz** [comment]
> > We could also add a Switch to Dark/White mode option.
> 
> So is that set to the system theme at the moment?

Yep

**Comment by vrindavan** [comment]
@adityakush24 are you going to take this up and add a toggle?

**Comment by adityakush24** [comment]
> @adityakush24 are you going to take this up and add a toggle?

Yes @vrindavan, I have added the toggle and updated the PR. Thank you.

**Comment by adityakush24** [comment]
> We could also add a Switch to Dark/White mode option.

@okyanusoz I have added the toggle and updated the PR. Thanks a lot for the suggestion.

**Comment by okyanusoz** [comment]
You’re welcome!

File: web/onload.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 $(function() {
 
   read_settings_from_cookie();
-
+
```
Comment (bitwiseman) [inline web/onload.js:5]:
```suggestion

```

**Pull request merged**